### DISCUSSION TITLE
We have issue forms!

### DISCUSSION THREAD

**Discussion by mcollina** [fastify/fastify#3051] [Ideas] [answered]
We can use "issue forms" instead of normal issues for some type of reports:

https://gh-community.github.io/issue-template-feedback/welcome/

![image](https://user-images.githubusercontent.com/52195/116709530-0dd11f80-a9d1-11eb-9e6a-32fe7b6e0503.png)

Here is also an example from imagemagick: https://github.com/ImageMagick/ImageMagick/issues/new?assignees=&labels=&template=bug-report.yml

WDYT, should we use them to improve the quality of the reports?

**Comment by jsumners** [comment]
It's worth a try.

**Reply by mcollina** [reply]
@Fdawgs would you like to take a stab?

**Reply by Fdawgs** [reply]
ImageMagick's example is pretty nice!

I'll give it a try and throw it into a draft PR

**Reply by zekth** [reply]
+1 on it

**Reply by Fdawgs** [reply]
See https://github.com/fastify/.github/pull/8 for bug report issue form PR. 😸 
Borrowed a section from ImageMagick's.

**Reply by Fdawgs** [reply]
As they've been in place for over 2 weeks now, what's the general consensus on the form so far?

I think having the mandatory fields is great for helping with issues, rather than having to ask each time what version of Node and Fastify a reporter is running on.

As it sits in the https://github.com/fastify/.github repo, which applies to all Fastify org repos, I think it needs an optional plugin version field to help with plugin repos.

**Reply by mcollina** [reply]
yes

**Reply by Eomm** [reply]
I was wondering if linking https://npm.runkit.com/fastify in the reproduce section may help people to produce better scripts

**Comment by Fdawgs** [comment, marked as answer]
Issues forms have been migrated from markdown to YAML based ones, with separate forms for the `website` repo:

https://github.com/fastify/.github/pull/8 
https://github.com/fastify/.github/pull/11
https://github.com/fastify/website/pull/281

**Reply by mcollina** [reply]
Thanks @Fdawgs !
